docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -1,4 +1,4 @@
-# Loris API - v0.0.0-dev
+# LORIS API - v0.0.0-dev
```

Comment:
Can you review the draft of the purpose/scope of the API defined #5826 and check if this falls within it?

I don't know enough about EEG to be able to tell if these endpoints are of general purpose usage for the scientific community or if they're just being added for the EEG module.

Reply:
@driusan They are of general purpose usage for the scientific community since it is probably the best way to grep/download EEG data given the size of the datasets. The EEG module does not depend on those endpoints.